bundles/AdminBundle/Resources/public/js/pimcore/settings/thumbnail/item.js

```diff
@@ -369,6 +369,44 @@ pimcore.settings.thumbnail.items = {
         }];
     },
 
+    dragdropRenderer: function (el) {
+        try {
+            // add dnd support
+            let dd = new Ext.dd.DropZone(el.getEl(), {
+                reference: this,
+                ddGroup: "element",
+                getTargetFromEvent: function (e) {
+                    return this.getEl();
+                }.bind(el),
+
+                onNodeOver: function (target, dd, e, data) {
+                    if (data.records.length === 1 && data.records[0].data.elementType === "asset") {
+                        return Ext.dd.DropZone.prototype.dropAllowed;
+                    }
+                },
+
+                onNodeDrop: function (target, dd, e, data) {
+                    if (pimcore.helpers.dragAndDropValidateSingleItem(data)) {
+                        var record = data.records[0];
+                        var data = record.data;
+
+                        if (data.elementType === "asset") {
```

Comment:
Same here, maybe change to 
```suggestion
                        if (data.type === "image") {
```
otherwise I get the icon that the drop is not allowed, but I am still able to drop the asset.